### DISCUSSION TITLE
Release 4.0.3 Board eingefroren

### DISCUSSION THREAD

**Discussion by decanyon** [dewenni/ESP_Buderus_KM271#93] [Q&A]
Mir ist aufgefallen dass sich das Board alle 10 - 20 Stunden aufhängt. Es ist nicht mehr erreichbar. Weder über WebUI noch sendet es über MQTT.
In diesem Fall leuchten die Grüne und orangene LED dann dauerhaft.

Hat das noch jemand beobachtet?

**Comment by dewenni** [comment]
Das kann ich selbst nicht bestätigen. Das hatte ich bei diesem Projekt generell so noch nicht. In der Version <4.0 hatte ich öfters mal das Problem, dass der Webserver nicht reagierte, aber die MQTT Verbindung war immer zuverlässig.
Mit der 4.0 und dem "neuen" Webserver läuft aber auch das bei mir sehr stabil.

Bei meiner Rolladensteuerung die auf einem ESP8266 läuft habe ich so was aber auch. Aber da passiert das höchstens 1x pro Monat.
Könnte also vielleicht ein Problem der Hardware sein?!

**Reply by solutionfound** [reply]
Gestern habe ich von Version 3.2.5 über das OTA Update auf die 4.0.3. gewechselt und es lief dann fast 3 Stunden ohne Probleme und dann ist das Board mit gleichen Symptomen wie hier beschrieben eingefroren.
Ich habe es dann nicht mehr außerhalb des Setup-Modus zum laufen bekommen. Auch ein flashen der gestern veröffentlichen 4.0.4 hat keinen Unterschied gemacht. Als letztes habe ich per Kabel nochmals das Update gemacht, aber auch das hat keinen Unterschied gebracht.
Da die Heizung 100km entfernt bei meiner Mutter ist, bin ich schweren Herzens wieder zurück auf die 3.2.5 gewechselt, da dann sofort und bis jetzt stabil läuft.

Was mit auch aufgefallen ist, dass manchmal im Setup-Mode während ich gewartet habe bis der Rechner den Access Point akzeptiert alle Lampen beim Board ausgingen. Eine Verbindung war dann auch nicht möglich. Vielleicht ist es auch im Setup-Mode abgestürzt. Das ist aber nicht immer der Fall gewesen nur so ca. in 25% der Fälle.
